dot/network/discovery.go

```diff
@@ -150,7 +150,8 @@ func (d *discovery) advertise() {
 
 			ttl, err = d.rd.Advertise(d.ctx, string(d.pid))
 			if err != nil {
-				logger.Debugf("failed to advertise in the DHT: %s", err)
+				// TODO: This fails consistently.
+				logger.Warnf("failed to advertise in the DHT: %s", err)
```

Comment:
i think we can probably remove the error log or change it do debug, as when you start the first node and it has no peers, this will always error

Reply:
> this will always error

For some time until another node is up right? Maybe we can have it as info with something less dramatic than `failed` such as

```suggestion
				logger.Infof("cannot advertise in the DHT: %s", err)
```

Reply:
> i think we can probably remove the error log or change it do debug, as when you start the first node and it has no peers, this will always error

I just tried a dev network with 3 nodes. I saw it only in the beginning.

I will remove my comment `This fails consistently`.

We `advertise` in the DHT at regular intervals, so we shouldn't remove this log. During the first time it says that `there are no peers in the table` (so anyone looking at the logs, would know to avoid it). This warning does not show after that.

I would rather have this warning in case this fails for any other reason, than have that error hidden in the debug mode.

Reply:
I'm fine to leave it as-is for now